docs/examples/plot_ND_basis_function.py

```diff
@@ -137,7 +137,7 @@
                  f"b_{{{basis_b_element}}}(x)$", color="b")
 
 for cc in range(len(basis_elem_idx)):
-    axs[cc].contourf(X, Y, Z[basis_elem_idx[cc]], cmap="Blues")
+    axs[cc].contourf(X, Y, Z[:, :, basis_elem_idx[cc]], cmap="Blues")
```

Comment:
```suggestion
    axs[cc].contourf(X, Y, Z[..., basis_elem_idx[cc]], cmap="Blues")
```
these are equivalent, but the `...` syntax is more robust to adding dimensions. just pointing it out, in case you aren't familiar with it

Reply:
There are other locations where this syntax could be helpful too, but I just noted it here.